Core/tooling/build.sh

```diff
@@ -10,15 +10,12 @@ mkdir -p build/final
 ./node_modules/.bin/tsc --project tsconfig.json
 
 # build raw/ with webpack
-./node_modules/.bin/webpack --config webpack.node.config.js
 ./node_modules/.bin/webpack --config webpack.bare.config.js
 
 # move modified raw/ to bundles/
 node tooling/fix-bundles.js
 
 # concatenate external deps into one bundle
-( cd build/bundles && cat node-html-sanitize-bundle.js node-emailjs-bundle.js node-openpgp-bundle.js > node-deps-bundle.js )  # node deps
-( cd build/bundles && cat node-html-sanitize-bundle.js node-emailjs-bundle.js node-dev-openpgp-bundle.js > node-dev-deps-bundle.js )  # node dev - for tests
```

Comment:
And this line should probably be retained too:

```
 ( cd build/bundles && cat node-html-sanitize-bundle.js node-emailjs-bundle.js node-dev-openpgp-bundle.js > node-dev-deps-bundle.js )  # node dev - for tests
```

But use updated code - without node instance